client/src/pages/Dashboard/EditProfilePage.tsx

```diff
@@ -0,0 +1,281 @@
+                    data-cy="user-phone"
+                    defaultCountry={"us"}
+                    {...formik.getFieldProps('phoneNumber')}
+                  />
+                  {formik.touched.phoneNumber && formik.errors.phoneNumber ? (
+                <div>{formik.errors.phoneNumber}</div>
+              ) : null}
+              </Grid>
+              <Grid item xs={3}>
+                <Button variant='outlined' color='secondary'> ADD NUMBER</Button>
+              </Grid>
+            </Grid>
+
+            <Grid container spacing={2} className={classes.gridcontainer}>
+              <Grid item xs={3}>
+                <Box component='div' p={1}>
+                  <Typography className={classes.typography} component='h1' variant='h6'>EMAIL ADDRESS</Typography>
+                </Box>
+              </Grid>
+              <Grid item xs={9}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                placeholder='john-doe@gmail.com'
+                id="email"
+                name="email"
+                {...formik.getFieldProps('email')}
+              />
+              {formik.touched.email && formik.errors.email ? (
+                <div>{formik.errors.email}</div>
+              ) : null}
+              </Grid>
+            </Grid>
+            <Grid container spacing={2} className={classes.gridcontainer}>
+              <Grid item xs={3}>
+                <Box component='div' p={1}>
+                  <Typography className={classes.typography} component='h1' variant='h6'>WHERE YOU LIVE</Typography>
+                </Box>
+              </Grid>
+              <Grid item xs={9}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                placeholder='Address'
+                name="address"
+                id="address"
+                {...formik.getFieldProps('address')}
+              />
+              </Grid>
+            </Grid>
+            <Grid container spacing={2} className={classes.gridcontainer}>
+              <Grid item xs={3}>
+                <Box component='div' p={1}>
+                  <Typography className={classes.typography} component='h1' variant='h6'>DESCRIBE YOURSELF</Typography>
+                </Box>
+              </Grid>
+              <Grid item xs={9}>
+                  <TextareaAutosize 
+                className={classes.textbox} 
+                aria-label="minimum height textarea" 
+                minRows={4} 
+                placeholder="About you" 
+                {...formik.getFieldProps('description')}
+                />
+              </Grid>
+            </Grid>
+          </Grid>
+          <Button
+            type="submit"
+            variant="contained"
+            color="secondary"
+            className={classes.submit}
+            xs={1}
+          >
+            SUBMIT
+          </Button>
+        
+      </div>
```

Comment:
Please fix spacing/indentation issues. 

You can install the prettier extension for your code editor and configure it to format on save (use the prettiererc file for config)

Reply:
done.